### DISCUSSION TITLE
GWSMO package not installing

### DISCUSSION THREAD

**Discussion by SimonYeo84** [chocolatey/home#156] [Q&A] [answered]
It seems as though the GWSMO package is broken with Chocolatey and keeps failing to install. Does anyone know why it is broken or if it will be getting fixed at all as I need to deploy it a number of new machines in my environment so I can keep it monitored and auto updated. Thanks

**Comment by TheCakeIsNaOH** [comment, marked as answer]
Taking a quick peek at the package, it seems like the package is outdated, so the checksum is incorrect. 

If a package is outdated, the first step is to contact the package maintainer, as per the package triage process:
https://docs.chocolatey.org/en-us/community-repository/users/package-triage-process#the-triage-process

As a workaround, you may be able to use `--checksum` and `--checksum64`, or alternatively `--ignore-checksums`, while going through the triage process.

**Reply by SimonYeo84** [reply]
So when adding in the --ignore-checksums, you reckon you can add it to the package like google-workspace-sync--ignore-checksums?

**Reply by TheCakeIsNaOH** [reply]
You add it like `choco install google-workspace-sync --ignore-checksums` or `choco upgrade google-workspace-sync --ignore-checksums`

**Reply by SimonYeo84** [reply]
That's if you do it via Commands in JumpCloud. Is it possible to do it via Software Management? That way I can manage it better and keep it up to date as I have an Auto Updater package in there for all Chocolatey apps

**Reply by TheCakeIsNaOH** [reply]
You would have to ask whoever in your support channels for JumpCloud about that.

**Reply by SimonYeo84** [reply]
Yea no worries. Will ask the question. Cheers for your comments so far. Appreciate it
